.github/workflows/renovate-config.yml

```diff
@@ -0,0 +1,19 @@
+name: renovate-config-validator
+
+on:
+  push:
+    paths:
+      - '.github/renovate.json5'
+    branches:
+      - main
+  pull_request:
+    paths:
+      - '.github/renovate.json5'
+  workflow_dispatch:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.2.0
+      - uses: suzuki-shunsuke/github-action-renovate-config-validator@v0.1.2
```

Comment:
I choose https://github.com/suzuki-shunsuke/github-action-renovate-config-validator, not 
https://github.com/rinchsan/renovate-config-validator by following reasons.

- too simple implementation
- support renovate.json5 without any configs
- https://github.com/suzuki-shunsuke#oss-development maintain too many terraform tools and [aqua](https://github.com/aquaproj/aqua) which is also upgrade by renovate

Reply:
It seems no other marketplace github actions for it. Otherwise we can do `apt install npm && npx --package renovate -c 'renovate-config-validator'`. Is this better? (for me both ok)

Reply:
I tried it and found that there is no reason to use marketplace action, so change it.
- https://github.com/runatlantis/atlantis/pull/2832/commits/985dd57c7b15c172db06ab52bd0b693eeb9eb644
- failed example job https://github.com/runatlantis/atlantis/actions/runs/3735307207/jobs/6338392964